http-plugins/src/main/java/io/cdap/plugin/batch/HTTPCallbackAction.java

```diff
@@ -148,18 +157,32 @@ public HttpRequestConf() {
     @SuppressWarnings("ConstantConditions")
     public void validate() {
```

Comment:
this shouldn't exist anymore right?

Reply:
Yes, we can remove the overridden method. Done